go/mysql/query.go

```diff
@@ -901,7 +901,13 @@ func (c *Conn) parseStmtArgs(data []byte, typ querypb.Type, pos int) (sqltypes.V
 		default:
 			return sqltypes.NULL, 0, false
 		}
-	case sqltypes.Decimal, sqltypes.Text, sqltypes.Blob, sqltypes.VarChar, sqltypes.VarBinary, sqltypes.Char,
+	case sqltypes.Char:
+		// NOTE: vitessio/vitess drops the sqltypes.Char information and returns the type as VarBinary, but
```

Comment:
Seems like we need more test to know whether this is a good idea or not, would be a good followup for you or someone else

Reply:
Yeah, it would be awesome to have more tests going through a running sql-server and exercising the Vitess code better.